SlevomatCodingStandard/Sniffs/Classes/AbstractPropertyAndConstantSpacingSniff.php

```diff
@@ -0,0 +1,102 @@
+<?php declare(strict_types = 1);
+
+namespace SlevomatCodingStandard\Sniffs\Classes;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use SlevomatCodingStandard\Helpers\TokenHelper;
+use function assert;
+use function in_array;
+use const T_COMMENT;
+use const T_CONST;
+use const T_DOC_COMMENT_OPEN_TAG;
+use const T_FUNCTION;
+use const T_PRIVATE;
+use const T_PROTECTED;
+use const T_PUBLIC;
+use const T_SEMICOLON;
+use const T_STRING;
+use const T_VAR;
+use const T_VARIABLE;
+
+abstract class AbstractPropertyAndConstantSpacingSniff implements Sniff
+{
+
+	/** @var int */
+	public $minLinesCountBeforeWithComment = 1;
+
+	/** @var int */
+	public $maxLinesCountBeforeWithComment = 1;
+
+	/** @var int */
+	public $minLinesCountBeforeWithoutComment = 0;
```

Comment:
Please set the default value to `1`.

Reply:
Sure, makes sense.

Reply:
You sure? It doesn't look that good to me ... and seems unnecessary.

```php
final class OperationalStatus extends Enum
{
    public const UP = 1;
    public const DOWN = 2;
    public const TESTING = 3;
    public const UNKNOWN = 4;
    public const DORMANT = 5;
    public const NOT_PRESENT = 6;
    public const LOWER_LAYER_DOWN = 7;
...
```
vs
```php
final class OperationalStatus extends Enum
{
    public const UP = 1;

    public const DOWN = 2;

    public const TESTING = 3;

    public const UNKNOWN = 4;

    public const DORMANT = 5;

    public const NOT_PRESENT = 6;

    public const LOWER_LAYER_DOWN = 7;

...
```

Reply:
You're right.